cmp/grid/impl/InitPersist.ts

```diff
@@ -32,6 +32,7 @@ export function initPersist(
         PersistenceProvider.create({
             persistOptions: {
                 path: `${path}.columns`,
+                persistDefaultValue: true,
```

Comment:
Implementation of Anslem's proposal:

> Set to false for grid columns here -

Note that this can be easily overriden by users using the PersistOption for persistColumns:
```
persistColumns: { persistDefaultValue: false }
```